FreeRTOS/Test/CMock/smp/multiple_priorities_no_timeslice/covg_multiple_priorities_no_timeslice_utest.c

```diff
@@ -38,58 +39,54 @@
 #include "queue.h"
 
 /* Test includes. */
-#include "unity.h"
-#include "unity_memory.h"
 #include "../global_vars.h"
 #include "../smp_utest_common.h"
+#include "unity.h"
+#include "unity_memory.h"
 
 /* Mock includes. */
-#include "mock_timers.h"
 #include "mock_fake_assert.h"
 #include "mock_fake_port.h"
+#include "mock_timers.h"
 
-#define taskTASK_YIELDING       ( TaskRunning_t ) ( -2 )
-
+#define taskTASK_YIELDING (TaskRunning_t)(-2)
+#define tskSTACK_FILL_BYTE (0xa5U)
 
 /* ===========================  EXTERN VARIABLES  =========================== */
+extern volatile UBaseType_t uxCurrentNumberOfTasks;
 extern volatile UBaseType_t uxDeletedTasksWaitingCleanUp;
 extern volatile UBaseType_t uxSchedulerSuspended;
-extern volatile TCB_t *  pxCurrentTCBs[ configNUMBER_OF_CORES ];
+extern volatile TCB_t *pxCurrentTCBs[configNUMBER_OF_CORES];
 extern volatile BaseType_t xSchedulerRunning;
 extern volatile TickType_t xTickCount;
+extern List_t xSuspendedTaskList;
+extern List_t xPendingReadyList;
+extern List_t pxReadyTasksLists[configMAX_PRIORITIES];
+extern volatile UBaseType_t uxTopReadyPriority;
+extern volatile BaseType_t xYieldPendings[configNUMBER_OF_CORES];
 
-/* ==============================  Global VARIABLES ============================== */
-TaskHandle_t xTaskHandles[configNUMBER_OF_CORES] = { NULL };
+/* ==============================  Global VARIABLES
+ * ============================== */
+TaskHandle_t xTaskHandles[configNUMBER_OF_CORES] = {NULL};
 
 /* ============================  Unity Fixtures  ============================ */
 /*! called before each testcase */
-void setUp( void )
-{
-    commonSetUp();
-}
+void setUp(void) { commonSetUp(); }
```

Comment:
Is this part of a standard code formatting tool?
I am not particularly fond of this formatting change.